contrib/node/src/python/pants/contrib/node/subsystems/resolvers/node_resolver_base.py

```diff
@@ -64,6 +64,8 @@ def _get_target_from_package_name(self, target, package_name, file_path):
 
     dep_spec_path = os.path.normpath(os.path.join(target.address.spec_path, address_path))
     for dep in target.dependencies:
+      if dep.is_synthetic:
```

Comment:
This... doesn't quite make sense to me. It feels like it will do the wrong thing if a target depends on multiple thrift libraries, since it's not actually matching the package name?

Reply:
Fixed: Synthetic targets shouldn't have been there